src/utils/getApplicableFormats.ts

```diff
@@ -3,6 +3,14 @@ import { RegularNode, SchemaNodeKind } from '@stoplight/json-schema-tree';
 import { COMMON_JSON_SCHEMA_AND_OAS_FORMATS } from '../consts';
 
 export function getApplicableFormats(schemaNode: RegularNode): [type: SchemaNodeKind, format: string] | null {
+  // Format is handled for contentMediaType type i.e. binary format
+  if (
+    schemaNode.fragment['contentMediaType'] === 'application/octet-stream' &&
+    schemaNode.types &&
+    schemaNode.types.length > 0
+  ) {
+    return [schemaNode.types[0], 'binary'];
+  }
```

Comment:
Should this check come after the `format === null` check?

Reply:
Yes, I have updated the file please check once .